scenes/globals/pause/pause_overlay.tscn

```diff
@@ -58,26 +58,34 @@ theme_type_variation = &"PlayerRibbon"
 layout_mode = 2
 text = "Game Paused"
 
+[node name="ResumeButton" type="Button" parent="TabContainer/PauseMenu/VBoxContainer"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 4
+theme_type_variation = &"FlatButton"
+text = "Resume"
+
 [node name="OptionsButton" type="Button" parent="TabContainer/PauseMenu/VBoxContainer"]
 unique_name_in_owner = true
 layout_mode = 2
 size_flags_horizontal = 4
 theme_type_variation = &"FlatButton"
 text = "Options"
 
-[node name="ResumeButton" type="Button" parent="TabContainer/PauseMenu/VBoxContainer"]
+[node name="TitleScreenButton" type="Button" parent="TabContainer/PauseMenu/VBoxContainer"]
 unique_name_in_owner = true
 layout_mode = 2
 size_flags_horizontal = 4
 theme_type_variation = &"FlatButton"
-text = "Resume"
+text = "Back to title screen"
```

Comment:
I'm not very sure about this label. Other option I handled: "Back to main menu", "Back".

Reply:
I don't think "Back" is a destructive-enough sounding word.

If we're going back to the title, "(Quit/Exit) to Title (Screen)" would read better to me. (I think these should be in Title Case.)

If we're going back to Fray's End then perhaps "Abandon Quest" would work?

Reply:
I checked in Spelunky 2 (other games are available, just by way of example).  It uses "Quit Game" on the main menu as the "return to the OS" action, and "Exit to Title" in the pause menu.